lib/oofatfs/ffunicode.c

```diff
@@ -499,6 +499,13 @@ DWORD ff_wtoupper ( /* Returns up-converted code point */
     DWORD uni       /* Unicode code point to be up-converted */
 )
 {
+    #if FF_FS_ASCII_UPPER_ONLY
```

Comment:
```suggestion
    #if FF_FS_CASE_INSENSITIVE_COMPARISON_UPPERCASE_ONLY
```
or `FF_FS_CASE_INSENSITIVE_UPPERCASE_ONLY` ,which is shorter